streamlit_dashboard/dashboard.py

```diff
@@ -10,11 +10,19 @@
 from bs4 import BeautifulSoup
 from database_connection import get_connection, get_cursor
 from streamlit_option_menu import option_menu
+import streamlit_shadcn_ui as ui
+from streamlit_card import card
+
+
+st.set_page_config(
+    layout="wide"
```

Comment:
Good idea to have the page set layout set to wide, makes the dashboard look better :)